dom.go

```diff
@@ -2,18 +2,44 @@ package vecty
 
 import (
 	"fmt"
+	"reflect"
 
 	"github.com/gopherjs/gopherjs/js"
 )
 
-// Core implements the Context method of the Component interface, and is the
-// core/central struct which all Component implementations should embed.
+var (
+	errMissingParent = fmt.Errorf("Missing parent node")
+	errEmptyElement  = fmt.Errorf("Empty element or node")
```

Comment:
Minor Go style issue, error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

Source: https://github.com/golang/go/wiki/CodeReviewComments#error-strings.